terraform/cloudfoundry/cf_api_elb.tf

```diff
@@ -22,3 +22,29 @@ resource "aws_elb" "cf_cc" {
     ssl_certificate_id = "${var.router_external_cert_arn}"
   }
 }
+
+resource "aws_elb" "cf_uaa" {
+  name = "${var.env}-cf-uaa"
+  subnets = ["${split(",", var.infra_subnet_ids)}"]
+  idle_timeout = "${var.elb_idle_timeout}"
+  cross_zone_load_balancing = "true"
+  security_groups = [
+    "${aws_security_group.cf_api_elb.id}",
+  ]
+
+  health_check {
+    target = "TCP:8080"
```

Comment:
I think if you hit any endpoint (/ or even /whatever) you get some html error page back - which indicates that the process is alive & it's an http response, so you can use http check. Not sure how much better than just TCP check it is...